default.py

```diff
@@ -111,7 +114,7 @@ def build_url(query):
 
 # Launch
 elif mode[0] == 'folder':
-    lutris = settings.getSetting('lutris_executable')
+    lutris = executable
```

Comment:
Ahh, right. Almost everything is in the scope of the `if mode is None:`. But I’m wondering, if the `executable` variable doesn’t exist, how can the `slug` and `args` variables exist? Are not they defined in the same scope as `executable`?